internal/errs/silencederror.go

```diff
@@ -5,14 +5,16 @@ import (
 )
 
 type silencedError struct {
-	error
+	err error
 }
 
-func Silence(err error) silencedError {
-	return silencedError{err}
+func Silence(err error) *silencedError {
+	return &silencedError{err}
 }
 
-func (s *silencedError) Unwrap() error { return s.error }
+func (s *silencedError) Error() string { return s.err.Error() }
```

Comment:
```suggestion
func (s *silencedError) Error() string { return "silenced error" }
```

This will tell us that it was funnelled through silencedError when we receive the error. Otherwise we just get the same error twice.

Reply:
Good call